src/components/embed_pages/FaceToFaceClassInfoEmbedPage.ts

```diff
@@ -0,0 +1,84 @@
+import { EmbedPage } from '../models/EmbedPage';
+import { client } from '../../index';
+import * as event from '../../../assets/event.json';
```

Comment:
Los json pueden importarse sin el asterisco
```suggestion
import event from '../../../assets/event.json';
```